src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package codeu.controller;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ *
+ * @author Agnieszka
+ */
+public class ActivityFeedServlet extends HttpServlet {
+    
+    @Override
+    public void init() throws ServletException{
```

Comment:
Be consistent with the spacing. Notice how in line 19 there is a space before the {. Here there is only one. Pick one and use that spacing for the entire file.